.github/workflows/rust.yaml

```diff
@@ -101,11 +101,26 @@ jobs:
     runs-on: windows-latest
     steps:
       - uses: actions/checkout@v2
+      - run: echo "VCPKG_ROOT=$env:VCPKG_INSTALLATION_ROOT" | Out-File -FilePath $env:GITHUB_ENV -Append
+      - run: vcpkg install openssl:x64-windows
```

Comment:
Here should be OK to change back https://github.com/sfackler/rust-openssl/blob/master/.github/workflows/ci.yml#L94